openslides/motions/models.py

```diff
@@ -771,6 +771,25 @@ class MotionChangeRecommendation(RESTModelMixin, models.Model):
     creation_time = models.DateTimeField(auto_now=True)
     """Time when the change recommendation was saved."""
 
+    def collides_with_other_recommendation(self):
+        recommendations = MotionChangeRecommendation.objects\
```

Comment:
You can use brackets around the if condition (e.g. [here](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/views.py#L163)) to avoid the backslash; the same three lines below.